papaya-web/src/main/java/pl/edu/agh/papaya/rest/projects/service/SprintsRestService.java

```diff
@@ -110,8 +105,56 @@
         return ResponseEntity.ok(sprintMapper.mapToApi(changedSprint));
     }
 
+    public ResponseEntity<SprintSummaryDto> getSprintSummary(Long projectId, Long sprintId) {
+        Sprint sprint = getValidSprint(sprintId);
+        Project project = projectsRestService.getValidProject(projectId);
+
+        if (project.isAdmin(userContext.getUser())) {
+            throw new ForbiddenAccessException();
+        }
+
+        List<UserAvailabilityDto> userAvailabilityDtos = availabilityRestService.getUserAvailabilities(sprintId);
+        Duration totalAvailableTime = sprintService.getTotalAvailableTimeBySprintId(sprintId);
```

Comment:
W tych dwóch linijkach się w sumie robi dwa razy to samo zapytanie na bazie pod spodem, więc to pewnie można by zrobić inaczej i już operować na wynikach jednego query, ale możemy tak żyć

Reply:
`getUserAvailabilities` -> `getUserAvailabilityDtos`?

Reply:
Poprawione, dodatkowo z wykorzystaniem obiektu `sprint`, który i tak już jest